client/src/rpc_client.rs

```diff
@@ -1,3 +1,12 @@
+//! Communication with a Solana node over RPC.
+//!
+//! Software that interacts with the Solana blockchain,
+//! whether querying its state or submitting transactions,
+//! communicates with a Solana node over [JSON-RPC],
+//! using the [`RpcClient`] type.
```

Comment:
It looks like your editor is set to line break around 60 characters or so?  In general I'd prefer an 80 character limit, for example:
```suggestion
//! Software that interacts with the Solana blockchain, whether querying its
//! state or submitting transactions, communicates with a Solana node over
//! [JSON-RPC], using the [`RpcClient`] type.
```
to reduce the number of comment lines.

Reply:
I'll reflow it.

It's not that I'm breaking at 60 chars, but that I'm breaking at single clauses within sentences. I find this easier to maintain and diff, but reading it in source is ugly. Just my habit.

Reply:
Done.